src/chemistry/geoschem/chemistry.F90

```diff
@@ -1613,16 +1617,20 @@ subroutine chem_init(phys_state, pbuf2d)
     IF ( Input_Opt%ITS_A_FULLCHEM_SIM .or. &
          Input_Opt%ITS_AN_AEROSOL_SIM ) THEN
        DO I = BEGCHUNK, ENDCHUNK
-          ! Restrict prints to one thread only
```

Comment:
Does removing this bring back the prints on all threads?

Reply:
No, prints are restricted to `Input_Opt%amIRoot` and `Init_Photolysis` runs once per CPU (on first chunk), so there is only one set of printouts.